policyengine_us/parameters/gov/usda/school_meals/categorical_eligibility.yaml

```diff
@@ -5,9 +5,16 @@ values:
     - snap # Supplemental Nutrition Assistance Program.
     - tanf # Temporary Assistance to Needy Families. Currently only available in CA and IL.
     - fdpir # Food Distribution Program on Indian Reservations.
+    - was_in_foster_care
+    - is_homeless
+    - is_runaway_child
+    - is_migratory_child
+    # Also Head Start child
 
 metadata:
   unit: program
   reference:
-    - title: Extending Categorical Eligibility to Additional Children in a Household
-      href: https://www.fns.usda.gov/cn/extending-categorical-eligibility-additional-children-household
+    - title: 7 CFR § 245.2 - Definitions
```

Comment:
could we keep the previous reference as well

Reply:
Max mentioned to replace it in the issue, should I keep or remove?

Reply:
lets keep then